# Generated by Django 3.0.2 on 2020-03-03 16:35

from django.db import migrations, models
import upload_validator


class Migration(migrations.Migration):

    dependencies = [
        ('hello', '0004_auto_20200301_2140'),
    ]

    operations = [
        migrations.AlterField(
            model_name='player',
            name='draws',
            field=models.IntegerField(default=0),
        ),
        migrations.AlterField(
            model_name='player',
            name='last_active',
            field=models.DateField(auto_now_add=True),
        ),
        migrations.AlterField(
            model_name='player',
            name='losses',
            field=models.IntegerField(default=0),
        ),
        migrations.AlterField(
            model_name='player',
            name='rating',
            field=models.IntegerField(default=3000),
        ),
        migrations.AlterField(
            model_name='player',
            name='rating_diff',
            field=models.IntegerField(default=0),
        ),
        migrations.AlterField(
            model_name='player',
            name='wins',
            field=models.IntegerField(default=0),
        ),
        migrations.AlterField(
            model_name='vegachessentry',
            name='entry',
            field=models.FileField(upload_to='hello.PictureWrapper/bytes/filename/mimetype', validators=[upload_validator.FileTypeValidator(allowed_types=['text/csv'])]),
        ),
    ]
